app/client/components/transferAccount/StandardTransferAccountList.tsx

```diff
@@ -236,7 +236,7 @@ class StandardTransferAccountList extends React.Component<
         onClick: (IdList: React.Key[]) => this.setApproval(false)
       },
       {
-        label: "New Bulk Transfer",
+        label: "New Bulk Disbursement",
```

Comment:
I think keeping it as "Transfer" makes sense, as there are also "RECLAMATION" and "BALANCE".

How about: `Create Bulk Transfer`?

Reply:
I was just going by the letter of the request, but this makes more sense. Fixed!